analog-nothing-dark/usr/share/asteroid-launcher/watchfaces/analog-nothing-dark.qml

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2022 - Timo Könnecke <github.com/eLtMosen>
```

Comment:
As the creator of this watchface, please add yourself on top of the contributors list you based your work on to ensure you are recognized as one of the contributors to the project.

Reply:
Added